### PR TITLE
Fixes #2405

### LINKED ISSUE #2405: PrintStringIntoString Not Inserting String Into Format
**Describe the bug**
A string is passed to the module PrintStringIntoString. The format is /%d/AF_%s%d.txt and the string being passed as input is Sum_Left_ACC_hdp.100

When executing the network, the string is not passed into the format, resulting in the output: /%d/AF_%d.txt

**To Reproduce**
Run the attached network and view the ReportStringInfo modules.
[Example.zip](https://github.com/SCIInstitute/SCIRun/files/9647752/Example.zip)

**Expected behavior**
The output string was expected to be /%d/AF_Sum_Left_ACC_hdp.100%d.txt

**Version**
v5.0-beta.Z.1 

**Desktop**
Windows 10 Pro

### PR DESCRIPTION
**Pull request by dcwhite**
I just had to move a block of code into the correct conditional branch. I bet PrintMatrixIntoString has the same problem, when multiple types of format specifiers are used in a format string.

### CODE REVIEW

**Comment by dcwhite** [comment]
@jessdtate Let's make this the final update for ZZZ release.

**Review by jessdtate — DISMISSED** [review]

File: src/Modules/Legacy/String/PrintStringIntoString.cc [hunk 1]
```diff
@@ -125,34 +120,33 @@ PrintStringIntoString::execute()
...
-                  currentstring = stringsH[inputport]; inputport++;
-                  if (currentstring)
+                  if (stringsH.size() == static_cast<size_t>(inputport))
+                  {
+                    lastport = true;
+                  }
+                  else
                   {
-                    str = currentstring->value();
+                    currentstring = stringsH[inputport]; inputport++;
```
Comment (GPayne) [inline src/Modules/Legacy/String/PrintStringIntoString.cc:141]: How about:
`currentstring = stringsH[inputport++];`

**Review by GPayne — APPROVED** [review]

File: src/Modules/Legacy/String/PrintStringIntoString.cc (hunk 1 above)
Comment (dcwhite) [inline src/Modules/Legacy/String/PrintStringIntoString.cc:141]:
```suggestion
                    currentstring = stringsH[inputport++];
```

Comment (dcwhite) [reply src/Modules/Legacy/String/PrintStringIntoString.cc]: Done. Now of course, Github needs you to re-approve.....grr

**Review by jessdtate — APPROVED** [review]

**Review by GPayne — APPROVED** [review]

**Pull request merged**